specification/marketplacenotifications/resource-manager/Microsoft.MarketplaceNotifications/stable/2021-03-03/MarketplaceNotifications.json

```diff
@@ -0,0 +1,256 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Blueprint API",
+    "description": "Microsoft Marketplace Blueprint API",
+    "version": "2021-03-03"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscription}/providers/Microsoft.MarketplaceNotifications/reviewsnotification": {
+      "get": {
+        "tags": [
+          "Notification"
+        ],
+        "operationId": "Notification_GetWithAuthorization",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/PrincipalId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Notification"
+            }
+          },
+          "default": {
+            "description": "Microsoft.MarketplaceNotifications error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetReviewsNotification": {
+            "$ref": "./examples/GetReviewsNotification.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.MarketplaceNotifications/operations": {
```

Comment:
must be marked x-ms-pageable  (https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls)